### DISCUSSION TITLE
error from phononLifetimes app

### DISCUSSION THREAD

**Discussion by matukumilli** [phoebe-team/phoebe#173] [Q&A] [answered]
Dear Phoebe developers,

Please help me with the below error from recent develop-version of Phoebe:

Launching App "phononLifetimes".

Error!
Input variable points path extrema hasn't been found in input

Regards,
Prasad
[phononLifetimes.txt](https://github.com/mir-group/phoebe/files/10422854/phononLifetimes.txt)

**Comment by jcoulter12** [comment]
Hi Prasad, 

Thanks for using Phoebe and letting us know you're having trouble with this! 
I just ran your input file with my development branch copy of Phoebe, and it runs without issue. 

To determine where your issue comes from, would you mind checking for me:
1) Run "git pull" in your phoebe repository directory, to double check that it's up to date with develop? 
2) Are you sure this is exactly the input file you're using? For example, did you clean up any comments, or change anything no matter how small, before sending it to me?

Some additional notes of advice for best performance, if you're trying to run a 2D material, you might see a reduced computational cost from using [dimensionality](https://phoebe.readthedocs.io/en/develop/inputFiles.html#dimensionality) = 2 in your input file. [useSymmetries](https://phoebe.readthedocs.io/en/develop/inputFiles.html#dimensionality) = true can also speed things up a great deal. If you go to lower temperatures, I would also recommend [windowType](https://phoebe.readthedocs.io/en/develop/inputFiles.html#windowtype) = "population"

Let me know these additional 1) and 2) items and we can work from there, 
Jenny

**Reply by jcoulter12** [reply]
Additionally, it would be great to have your output file (as in what outputs from the executable during a calculation) so I can see if anything strange has happened to the inputs during read-in.

**Comment by matukumilli** [comment]
Hi Jenney,

Yes I sent you a cleaned up input file.
Now I tried with a cleaned up input file, and it run without any error.  How come commented lines causing the difference?

input file with commented lines:
[phononLifetimes.txt](https://github.com/mir-group/phoebe/files/10434216/phononLifetimes.txt)

Thanks for your advice for better performance. I'll try them.

Regards,
Prasad

**Reply by jcoulter12** [reply]
Hi Prasad, 

Thanks for including the commented file. Interestingly, I find that this also runs correctly with the develop branch version of Phoebe. 

My concern about the comments was actually connected to my first question from the last message -- are you sure you're using the develop branch? Last week, I made a [pull request that had some fixes](https://github.com/mir-group/phoebe/pull/171) to Phoebe's ability to ignore comment lines in input files. If you're truly using the develop branch, that means my PR might have introduced a new issue. If you're not using the develop branch, perhaps this PR from last week will fix your issue. 

One more  comment on this input file -- putting these two variable names in doesn't do anything anymore: 
```
 dispFCFileName = "../disp_fc3.yaml"
# dispFC2FileName = "disp_fc2.yaml"
```
It won't cause problems either, but we simplified the inputs to just use `phono3py_disp.yaml` in September. I only comment on this because it might indicate you have an old copy of Phoebe. 

Thanks,
Jenny

**Comment by jcoulter12** [comment, marked as answer]
Hi Prasad,

Just an update -- I ran into this issue with comments causing the point path variable block to be missed or read in incorrectly. I'm about to merge PR https://github.com/mir-group/phoebe/pull/174 to develop, it should fix this issue. It was indeed introduced in PR https://github.com/mir-group/phoebe/pull/171 last week.

Thanks very much for reporting it, we rely on helpful users to do this!
Best, and let me know if this issue persists.
Jenny
